types/scene/wlr_scene.c

```diff
@@ -39,6 +40,13 @@
 #define DMABUF_FEEDBACK_DEBOUNCE_FRAMES  30
 #define HIGHLIGHT_DAMAGE_FADEOUT_TIME   250
 
+#define SCENE_BUFFER_SHOULD_BLUR(scene_buffer, scene) \
+	scene_buffer->backdrop_blur && is_scene_blur_enabled(&scene->blur_data) && \
+		(!scene_buffer->buffer_is_opaque || scene_buffer->opacity < 1.0f)
```

Comment:
Is it possible for the buffer to be opaque and the opacity to be less than 1? Haven't checked but not sure if is opaque is for the texture we are given